boards/mikroe/quail/doc/mikroe_quail.rst

```diff
@@ -0,0 +1,99 @@
+.. zephyr:board:: mikroe_quail
+
+Overview
+********
+MikroE Quail for STM32 is a development board containing an `STM32F427`_
+microcontroller. It is equipped with four mikrobus sockets.
+The edges of the board are lined with screw terminals and USB ports for
+additional connectivity.
+
+Hardware
+********
+The Quail board contains the following connections:
+
+  - Four MikroBUS connectors
+  - lined with 32 screw terminals and USB ports for
```

Comment:
Change to

"""
- 32 screw terminals
- two USB ports, one for programming and one for external storage
"""